SPECS/xapi.spec

```diff
@@ -26,6 +26,7 @@ Patch1007: xapi-1.249.32-update-schema-hash.XCP-ng.patch
 # Contributed upstream, can be dropped in next version bump
 Patch1010: xapi-1.249.32-allow-a-user-to-select-on-which-SR-to-run-quicktest.backport.patch
 Patch1011: xapi-1.249.32-redirect-fileserver-https.backport.patch
+Patch1012: xapi-1.249.32-add-vdi_update-filter-to-some-tests.backport.patch
```

Comment:
I'd regroup the two quicktest-related patches

Reply:
My comment could be interpreted in several ways, so: I don't mean merging the two patches together. Just listing them one next to the other.

Reply:
I agree and it's done